dlp/snippets/inspect/inspect_test.go

```diff
@@ -513,3 +513,24 @@ func TestInspectImageFile(t *testing.T) {
 		t.Errorf("TestInspectImageFile got %q, want %q", got, want)
 	}
 }
+
+func TestInspectImageFileAllInfoTypes(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	inputPath := "testdata/image.jpg"
+
+	var buf bytes.Buffer
+	if err := inspectImageFileAllInfoTypes(&buf, tc.ProjectID, inputPath); err != nil {
+		t.Errorf("inspectImageFileAllInfoTypes: %v", err)
+	}
+	got := buf.String()
+	if want := "Info type: DATE"; !strings.Contains(got, want) {
+		t.Errorf("inspectImageFileAllInfoTypes got %q, want %q", got, want)
+	}
+	if want := "Info type: PHONE_NUMBER"; !strings.Contains(got, want) {
+		t.Errorf("InspectBigQueryTableWithSampling got %q, want %q", got, want)
```

Comment:
Should be inspectImageFileAllInfoTypes not BQ sampling. Please fix.

Reply:
Apologies, Updated it. will take care that such things does't happen in the future.